### DISCUSSION TITLE
纯小白帮助文档

### DISCUSSION THREAD

**Discussion by Scorpions-09** [tw93/Pake#209] [Q&A]
看到版本已更新，小白的帮助文档希望也更新一下，尤其是下面这步，和帮助文档不相符，希望更新（希望知道同时构建多个软件时又该怎么写），还望大佬们帮助。
![image](https://user-images.githubusercontent.com/53962545/210170289-0c1d827a-c21a-4a76-bc6f-4880cec90223.png)

**Comment by Tlntin** [comment]
收到，晚点更新。

**Reply by Tlntin** [reply]
[wike](https://github.com/tw93/Pake/wiki/GitHub-Actions-%E5%9C%A8%E7%BA%BF%E7%BC%96%E8%AF%91%E5%A4%9A%E7%B3%BB%E7%BB%9F%E7%89%88%E6%9C%AC)
@Scorpions-09 已更新。
多个软件直接改app.csv就行了，推荐你用wps打开。你截图的这一步，理论上不需要修改。
